BUILD.md

```diff
@@ -449,6 +450,31 @@ export LD_LIBRARY_PATH=~/ComputeLibrary/build/
 
 ---
 
+### ArmNN
+See more information on the ArmNN Execution Provider [here](./docs/execution_providers/ArmNN-ExecutionProvider.md).
+
+#### Prerequisites
+* Supported backend: i.MX8QM Armv8 CPUs
+* Supported BSP: i.MX8QM BSP
+  * Install i.MX8QM BSP: `source fsl-imx-xwayland-glibc-x86_64-fsl-image-qt5-aarch64-toolchain-4*.sh`
+* Set up the build environment
+```
+source /opt/fsl-imx-xwayland/4.*/environment-setup-aarch64-poky-linux
+alias cmake="/usr/bin/cmake -DCMAKE_TOOLCHAIN_FILE=$OECORE_NATIVE_SYSROOT/usr/share/cmake/OEToolchainConfig.cmake"
+```
+* See [Build ARM](#ARM) below for information on building for ARM devices
+
+#### Build Instructions
+```
+./build.sh --use_armnn
+```
+The Relu operator is set by default to use the CPU execution provider for better performance. To use the ArmNN implementation build with --armnn_relu flag
```

Comment:
Is there a case when ArmNN implementation of Relu is useful?

Reply:
At the moment, the implementation with CPU offers a better performance but we let the use of the flag for future improvements.